<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>APSGDlg</class>
 <widget class="QDialog" name="APSGDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>471</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 1px  solid #000;</string>
   </property>
   <property name="title">
    <string>AutoIris</string>
   </property>
   <widget class="QCheckBox" name="cbAutoIris">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>21</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 0px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAISave">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>416</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>201</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string>Basic</string>
    </property>
    <widget class="QLineEdit" name="leAIShutterStep">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>210</y>
       <width>80</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px;
</string>
     </property>
     <property name="text">
      <string>Min Gain</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_33">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px;
</string>
     </property>
     <property name="text">
      <string>BR Avg Count</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px;
</string>
     </property>
     <property name="text">
      <string>Debug Mode</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leAIMaxShutter">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>80</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leAIBRRange">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>80</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px;
</string>
     </property>
     <property name="text">
      <string>BR Range</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_34">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px;
</string>
     </property>
     <property name="text">
      <string>Standard BR</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px;
</string>
     </property>
     <property name="text">
      <string>Max Gain</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px;
</string>
     </property>
     <property name="text">
      <string>Gain Step</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px;
</string>
     </property>
     <property name="text">
      <string>Max Shutter</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leAIMinShutter">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>180</y>
       <width>80</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;
</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbAIDebug">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>340</y>
       <width>21</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 0px</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px;
</string>
     </property>
     <property name="text">
      <string>Shutter Step</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leAIMinGain">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>270</y>
       <width>80</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px;
</string>
     </property>
     <property name="text">
      <string>Min Shutter</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leAIGainStep">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>300</y>
       <width>80</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leAIMaxGain">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>240</y>
       <width>80</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leAIBRAvgCount">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>80</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leAIStandardBR">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>80</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leAIBRInterval">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>80</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_37">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px;
</string>
     </property>
     <property name="text">
      <string>BR Interval</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>231</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Reflection/BackLight</string>
    </property>
    <widget class="QLineEdit" name="leAIMinPlateBR">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>180</y>
       <width>80</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_35">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px;
</string>
     </property>
     <property name="text">
      <string>Mode</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_36">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px;
</string>
     </property>
     <property name="text">
      <string>Min BR</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leAIBRStep">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>80</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leAIMaxBR">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>80</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_38">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px;
</string>
     </property>
     <property name="text">
      <string>Max BR</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_40">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px;
</string>
     </property>
     <property name="text">
      <string>BR Change Count</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_42">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px;
</string>
     </property>
     <property name="text">
      <string>BR Step</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leAIMaxPlateBR">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>80</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;
</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbAIReflection">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 0px</string>
     </property>
     <property name="text">
      <string>Reflection</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_43">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px;
</string>
     </property>
     <property name="text">
      <string>Min Plate BR</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_44">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px;
</string>
     </property>
     <property name="text">
      <string>Max Plate BR</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leAIBRChangeCount">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>210</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leAIMinBR">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>80</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;
</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbAIBackLight">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 0px</string>
     </property>
     <property name="text">
      <string>BackLight</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbAIReflectBackMode">
     <property name="geometry">
      <rect>
       <x>64</x>
       <y>240</y>
       <width>131</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>320</y>
      <width>231</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>No Recog</string>
    </property>
    <widget class="QLineEdit" name="leAINRMinBR">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>80</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_55">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px;
</string>
     </property>
     <property name="text">
      <string>Min BR</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_56">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px;
</string>
     </property>
     <property name="text">
      <string>Max BR</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leAINRMaxBR">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>80</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;
</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbAINoRecog">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>21</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 0px</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>471</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 1px  solid #000;</string>
   </property>
   <property name="title">
    <string>Light</string>
   </property>
   <widget class="QLineEdit" name="leLightOnGain">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>45</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_57">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px;
</string>
    </property>
    <property name="text">
     <string>ON Gain</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_58">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>15</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px;
</string>
    </property>
    <property name="text">
     <string>Shutter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leLightShutter">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbLight24H">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 0px</string>
    </property>
    <property name="text">
     <string>24H Light </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leLightOffGain">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>45</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_59">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px;
</string>
    </property>
    <property name="text">
     <string>OFF Gain</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLightSave">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
